index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p>Hello Microverse!</p>
```

Comment:
- [ ] Nice clean HTML 👍 , however, I think for the heading it would be better if you use the ``h1`` tag, rather than ``p``, as the ``p`` tag  is used for paragraphs and the ``h1``tag is used for the most important heading. You could read this for more information [semantic tags](https://www.teamscs.com/2017/09/html5-introduction-new-semanticstructural-elements/)